VRCXDiscordTracker/Core/AppConstants.cs

```diff
@@ -13,8 +13,23 @@ internal class AppConstants
     /// </summary>
     public static readonly Version AppVersion = Assembly.GetExecutingAssembly().GetName().Version ?? new Version(0, 0, 0);
 
+    /// <summary>
+    /// アプリケーションバージョンの文字列
+    /// </summary>
+    public static readonly string AppVersionString = AppVersion.ToString(3); // Major.Minor.Patch
```

Comment:
Updaterの`AppConstants`と同様、バージョン情報は文字列しか使用していないのでひとまとめにしてもいいかもしれません。

```suggestion
    /// <summary>
    /// アプリケーションバージョンの文字列
    /// </summary>
    public static readonly string AppVersionString = (Assembly.GetExecutingAssembly().GetName().Version ?? new Version(0, 0, 0)).ToString(3); // Major.Minor.Patch
```